datasets/hindiDiscourse/README.md

```diff
@@ -0,0 +1,98 @@
+# hindi-discourse
```

Comment:
nice ! thanks for providing all this informartion abou the dataset.

Can you change the format to match the dataset card formart we use in the template, as yacine mentioned ?

Reply:
yes sure, will do so